server/src/messageHandler.mjs

```diff
@@ -171,23 +146,24 @@ async function handleMessage(message, userId, ws) {
   }
 
   // Fire post trigger if there is one for this method
-  if ( oMsg.method in triggers ) {
-    if ( 'post' in triggers[oMsg.method] ) {
+  if ( oMsg.method in methodTriggers ) {
+    if ( 'post' in methodTriggers[oMsg.method] ) {
       try {
         const ctx = {
           logger: logger,
           setTimeout: setTimeout,
           setInterval: setInterval,
           sendEvent: function(onMethod, result, msg) {
-            const id = events.getRegisteredEventListener(onMethod);
-            if ( ! id ) { return; }
-            emit(id, result, msg, ws);
+            function fSuccess() {}
+            function fErr() {}
```

Comment:
Same comment as above

Reply:
added logger statements in fSuccess, fErr and fFatalErr